### PR TITLE
fix tsconfig

### LINKED ISSUE #26: Fix Tsconfigs
Adding yarn workspaces needs an update to our tsconfigs

### PR DESCRIPTION
**Pull request by wpank**
fixes various tsconfig options accross packages

closes #26

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -23,11 +22,14 @@
...
   },
-  "include": [],
+  "include": ["./node_modules/@trustwallet"],
   "references": [
     { "path": "./packages/ui" },
-    { "path": "./packages/walletconnect" }
+    { "path": "./packages/walletconnect" },
+    { "path": "./packages/provider" },
+    { "path": "./packages/types" },
+    { "path": "./packages/eslint-config" }
```
Comment (vikmeup) [inline tsconfig.json]: Is there a way to specify as `./packages/**` ?

File: packages/walletconnect/index.d.ts [hunk 2]
```diff
@@ -1 +0,0 @@
-declare module '@trustwallet/walletconnect'
```
Comment (vikmeup) [inline packages/walletconnect/index.d.ts]: I thought this file was needed to use walletconnect package, any chance to double check?

Comment (wpank) [reply packages/walletconnect/index.d.ts]: This gets added back in a later commit. Some of the changes in these first two commits may not make 100% sense

Comment (vikmeup) [reply packages/walletconnect/index.d.ts]: 🚀 then

File: tsconfig.json (hunk 1 above)
Comment (wpank) [reply tsconfig.json]: I'd rather leave it like this now. Can change it later. For example we don't want the `./packages/examples` in this

**Pull request merged**